### PR TITLE
[54]Add Page Navigation to your Pages

### PR DESCRIPTION
**Pull request by DamisiM**
## Task: `[54]Add Page Navigation to your Pages`
- `Updated html form task`
IMG_20201029_224642

### CODE REVIEW

**Review by adaugochi — CHANGES_REQUESTED** [review]
You don't have to create another copy of `productPage` again inside the `html-forms` folder. This file already exists in the `html` folder. Kindly remove it

File: submissions/damisiMahmood/html-forms/contactPage.html [hunk 1]
```diff
@@ -13,7 +13,8 @@ <h1> I-DAMSELS ELITE SHOPPING </h1>
                 <li><a href="#"> HOME</a></li> 
                 <li><a href="#">ABOUT</a></li> 
                 <li><a href="#">CATGORIES</a></li> 
-                <li><a href="#">BRANDS</a></li> 
+                <li><a href="productPage.html">PRODUCTS</a></li>
```
Comment (adaugochi) [inline submissions/damisiMahmood/html-forms/contactPage.html:16]: Are you sure this `href="productpage.html"` works? Try clicking on that link

**Review by adaugochi — CHANGES_REQUESTED** [review]

**Comment by DamisiM** [comment]
Made, relevant changes. Thank you

File: submissions/damisiMahmood/html-forms/contactPage.html [hunk 2]
```diff
@@ -13,7 +13,8 @@ <h1> I-DAMSELS ELITE SHOPPING </h1>
                 <li><a href="#"> HOME</a></li> 
                 <li><a href="#">ABOUT</a></li> 
                 <li><a href="#">CATGORIES</a></li> 
-                <li><a href="#">BRANDS</a></li> 
+                <li><a href="../html/productPage.html">PRODUCTS</a></li>
```
Comment (adaugochi) [inline submissions/damisiMahmood/html-forms/contactPage.html:16]: Your html folder is in the capital letters (HTML) but here you are smaller letter (html). Are you sure this PR you are pushing for review works properly?

Comment (adaugochi) [reply submissions/damisiMahmood/html-forms/contactPage.html:16]: Same goes for other places where you make the same mistake

**Review by adaugochi — CHANGES_REQUESTED** [review]

File: submissions/damisiMahmood/html-forms/contactPage.html (hunk 2 above)
Comment (DamisiM) [reply submissions/damisiMahmood/html-forms/contactPage.html:16]: The PR works properly on different browsers, however I have changed the name of the folder to small letters

**Review by adaugochi — CHANGES_REQUESTED** [review]

**Review by adaugochi — APPROVED** [review]

**Pull request merged**